pool/pool_manager.gno

```diff
@@ -174,7 +170,7 @@ func CreatePool(
 	}
 
 	// TODO: make this as a parameter
```

Comment:
```suggestion	
```

I really don't get this comments. IMHO, it doesn't need to be get from parameter

Reply:
@r3v4s 
https://github.com/gnoswap-labs/gnoswap/pull/442/commits/bb7be92dc62d69ea2f5dd69327ec547198c6bac1